docs/querying-data.rst

```diff
@@ -0,0 +1,153 @@
+Querying data
+=============
+
+Before you begin, read the :doc:`index` and :doc:`data-model` pages to learn about how data is stored in Kingfisher Process.
+
+Since most analysis is much easier to perform on compiled releases, we recommend working with compiled release collections to begin with.
+
+Get a list of compiled release collections from a particular source
+-------------------------------------------------------------------
+
+The following query returns a list of compiled release collections downloaded from the `State Procurement Agency of Georgia's OCDS API <https://odapi.spa.ge/>`__:
+
+.. code-block:: sql
+
+  select
+      *
+  from
+      collection --the `collection` table contains a list of all collections in the database
+  where
+      source_id = 'georgia_releases' --filter by collections from the 'georgia_releases' source.
+  and
+      cached_compiled_releases_count > 0 --filter by collections containing compiled releases
+  order by
+      id desc; --collection ids are sequential, order by newest first
+
+To find collections from a different source, change the ``source_id`` parameter. The ``source_id`` in Kingfisher Process is based on the name of the spider in Kingfisher Scrape.
+
+See the list of spiders in the `Kingfisher Scrape Github Repository <https://github.com/open-contracting/kingfisher-scrape/tree/master/kingfisher_scrapy/spiders>`__ for a list of possible sources. Each ``.py`` file is a spider, and the part before the ``.py`` extension is the ``source_id``.
+
+Get the JSON data stored in a collection
+----------------------------------------
+
+Use the collection ``id`` returned by the previous query to restrict your analysis to a single collection.
+
+The following query returns the full JSON data for the first 3 compiled releases in collection 584:
+
+.. code-block:: sql
+
+  select
+      data
+  from
+      data --raw OCDS JSON data is stored as jsonb blobs in the `data` column of the `data` table
+  join
+      compiled_release
+  on
+    data.id = compiled_release.data_id -- join to the `compiled_release` table to filter data from a specific collection
+  where
+      collection_id = 584
+  limit 3;
+
+To get data from a different collection, change the ``collection_id`` parameter.
+
+To get data from a collection containing releases or records, join to the ``release`` or ``record`` tables rather than the ``compiled_release`` table.
+
+.. tip:: Rendering JSON using Redash
+
+  If you are using Redash, you can render the results of a query as pretty printed and collapsible JSON by clicking the '+ New Visualization' button, setting the visualization type to 'table' and setting the data column to display as JSON.
+
```

Comment:
Very clear and very good section. The Redash section made sense as a helpdesk analyst but might feel out of place to an external person. I don't think it's high-priority but if we're targeting external users then this could be reframed around "querying tools, such as Redash <link>" can often render the results of a query…".

But as an analyst who uses redash this is very useful to me and I wouldn't want to lose that.

Reply:
Either that or we explicitly recommend running a redash instance earlier in the documentation?

Reply:
Tweaked slightly in https://github.com/open-contracting/kingfisher-process/pull/220/commits/a67316d7c276903bf568689e79ca957df34b5bb2 to make it clearer why Redash is mentioned